tests/testthat/test-epi_shift_internal.R

```diff
@@ -0,0 +1,65 @@
+library(dplyr)
+library(epiprocess)
+library(parsnip)
+library(workflows)
+
```

Comment:
Maybe I didn't understand the new function clearly, but isn't `epi_shift_internal` or `epi_shift_new` the new function that automatically creates lags or leads based on the role of the variable? The idea is to unify the functions `step_epi_ahead` and `step_epi_lag` and test that unified functions, right?

Reply:
Yes, and we had step_epi_lag_new, step_epi_ahead_new, etc. which I also merged.